tracing-appender/src/rolling/builder.rs

```diff
@@ -181,6 +183,33 @@ impl Builder {
         Self { suffix, ..self }
     }
 
+    /// Keep the last `n` log entries on disk.
+    ///
```

Comment:
maybe we should also add something like this, just to make sure the behavior is clear:
```suggestion
    ///
    /// When a new log file is created, if there are `n` or more
    /// existing log files in the directory, the oldest will be deleted.
```